src/validate/editor.js

```diff
@@ -0,0 +1,20 @@
+'use babel'
+
+import throwIfFail from './throw'
+
+export const isValidPoint = (textBuffer, line, column) => {
+  // Clip the given point to a valid one
+  const validPoint = textBuffer.clipPosition([line, column])
+  // Compare to original
+  return validPoint.isEqual([line, column])
+}
+
+export const throwIfInvalidPoint = (tb, line, col) => throwIfFail(
```

Comment:
What's a `tb`? `textBuffer`?

Reply:
For utility functions like this just expand variables like `col` to `column`, they are only typed twice and it makes things perfectly clear.

Reply:
No problem